modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Fields.tsx

```diff
@@ -76,8 +76,7 @@ const getRendererLabel = ({
 
 	if (internalRenderer?.label) {
 		return internalRenderer.label;
-	}
-	else {
+	} else {
```

Comment:
This will make SF upset. You can check with `ant format-source` from `./portal-impl` or run `yarn format` from `./modules`